app/components/previews/provider_interface/status_box_component_preview.rb

```diff
@@ -0,0 +1,59 @@
+module ProviderInterface
+  class StatusBoxComponentPreview < ActionView::Component::Preview
+    def awaiting_provider_decision
+      render_component_for ApplicationChoice.find_by(status: :awaiting_provider_decision)
```

Comment:
https://apply-for-te-1635-imple-g8dt6g.herokuapp.com/rails/components/provider_interface/status_box_component/awaiting_provider_decision gives a 500 for me.